azure-keyvault/azure/keyvault/secrets/_models.py

```diff
@@ -12,6 +12,17 @@
 
 
 class SecretAttributes(object):
+    """A secret's id and attributes.
+
+    Variables are only populated by the server, and will be ignored when
+    sending a request.
+
+    :param attributes: The secret management attributes
+    :type attributes: ~azure.keyvault.secrets._models.SecretAttributes
```

Comment:
What should be the correct type for this param?
Also, do we still want to document this?

Reply:
`azure.keyvault.secrets._models.SecretAttributes` is this class itself. The `attributes` parameter is of the generated model class also named `SecretAttributes`, which would be `azure.keyvault.secrets._generated.v7_0.models.SecretAttributes`. But we don't want to document the parameters because users don't need to know how to instantiate these models and shouldn't be thinking about that at all. Docs for these models need only describe them and their properties.